crypto/objects/objects.txt

```diff
@@ -1391,8 +1391,16 @@ id-tc26-gost-28147-constants 1	: id-tc26-gost-28147-param-Z : GOST 28147-89 TC26
 member-body 643 3 131 1 1	: INN	: INN
 member-body 643 100 1		: OGRN	: OGRN
 member-body 643 100 3		: SNILS	: SNILS
+member-body 643 100 5		: OGRNIP	: OGRNIP
 member-body 643 100 111	: subjectSignTool	: Signing Tool of Subject
 member-body 643 100 112	: issuerSignTool	: Signing Tool of Issuer
+member-body 643 100 113  : classSignTool   : Class of Signing Tool
```

Comment:
Is there a reference to some standard we could add as a comment here to explain where these come from?

Reply:
Well, the most relevant link seems to be http://www.garant.ru/products/ipo/prime/doc/70033464/

Reply:
Is that link likely to be stable? If so add that - if not perhaps just a sentence or so on what these OIDS are for.

Reply:
Added to the pull request. It should be rather stable.

Reply:
Did you push? I don't see it?

Reply:
Well. Got totally messed.
The ```git commit --amend``` does weird things. And when I try to checkout a new branch, it contains OCSP fixes, so I have no ideas.

Reply:
See #8309